components/VerifyOptions.yml

```diff
@@ -17,12 +17,6 @@ components:
       additionalProperties: false
       description: Options for specifying how the LinkedDataProof is verified.
       properties:
-        verificationMethod:
-          type: string
-          description: The URI of the verificationMethod used for the proof. If omitted, a default verification method will be used.
-        proofPurpose:
-          type: string
-          description: The purpose of the proof. Default 'assertionMethod'.
```

Comment:
I'm suggesting we remove this as well. You shouldn't be able to override `verificationMethod` or `proofPurpose` when verifying.